docs/source/author/topics.rst

```diff
@@ -613,11 +613,14 @@ Curriculum Areas Configuration File
 
       - ``name:`` The name of the curriculum area (this is what will be displayed to the
         user).
+      - ``colour:`` The CSS colour class to use for colouring the curriculum
+        area badge on the website.
+        This colour is also applied to all children of curriculum area.
```

Comment:
This is a bit of a chicken and egg situation, so will do my best to explain it.

Originally I didn't have the colour specified, and had CSS classes written to pick out based off the area's `slug`, however this would fail with different translations.

Next option was to let the author specify the colour in the configuration file using a hexadecimal code, or something similar, however you would end up with colours that don't match the rest of the website's branding. I didn't want to the responsibility of web design on the authors.

So I chose to specify a CSS class, that is defined in the SCSS files and uses colours from our branding colour palette. When translated, these values should be left how they are, similar to numerical values for lessons. A list of available colours could be defined in the documentation under this configuration file? I only added enough colours to cover all these groups (but don't know how final this list is), so eventually I should add probably twice as many.

I couldn't figure out a better solution, but this seemed to work so far.